<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="setWidget" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>550</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>550</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QGroupBox" name="SignalGroupBox">
            <property name="title">
             <string>串口设置</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="6,1,1">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="bottomMargin">
              <number>9</number>
             </property>
             <item>
              <layout class="QFormLayout" name="formLayout">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <property name="fieldGrowthPolicy">
                <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
               </property>
               <property name="rowWrapPolicy">
                <enum>QFormLayout::WrapLongRows</enum>
               </property>
               <property name="labelAlignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
               </property>
               <property name="formAlignment">
                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
               </property>
               <property name="horizontalSpacing">
                <number>25</number>
               </property>
               <property name="verticalSpacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item row="0" column="0">
                <widget class="QLabel" name="portLabel">
                 <property name="text">
                  <string>串口</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="1">
                <widget class="QComboBox" name="Comb_serialPort">
                 <item>
                  <property name="text">
                   <string>COM1</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM2</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>COM3</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QLabel" name="BaudRateLabel">
                 <property name="text">
                  <string>波特率</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <widget class="QComboBox" name="Comb_baudRate">
                 <item>
                  <property name="text">
                   <string>9600</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>19200</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>38400</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="2" column="0">
                <widget class="QLabel" name="DataBitsLabel">
                 <property name="text">
                  <string>数据位</string>
                 </property>
                </widget>
               </item>
               <item row="2" column="1">
                <widget class="QComboBox" name="Comb_dataBits">
                 <item>
                  <property name="text">
                   <string>5 bits</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>6 bits</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>7 bits</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>8 bits</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="3" column="0">
                <widget class="QLabel" name="ParityBitsLabel">
                 <property name="text">
                  <string>校验位</string>
                 </property>
                </widget>
               </item>
               <item row="3" column="1">
                <widget class="QComboBox" name="Comb_parityBits">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>奇校验</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>偶校验</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>space parity</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>mark parity</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="4" column="0">
                <widget class="QLabel" name="StopLabel">
                 <property name="text">
                  <string>停止位</string>
                 </property>
                </widget>
               </item>
               <item row="4" column="1">
                <widget class="QComboBox" name="Comb_stopBits">
                 <item>
                  <property name="text">
                   <string>1.0 bit</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>1.5 bits</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>2.0 bits</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item row="5" column="0">
                <widget class="QLabel" name="flowContrlLabel">
                 <property name="text">
                  <string>流控制</string>
                 </property>
                </widget>
               </item>
               <item row="5" column="1">
                <widget class="QComboBox" name="Comb_flowContrl">
                 <item>
                  <property name="text">
                   <string>None</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>rts/cts</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>xon/xoff</string>
                  </property>
                 </item>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="scanSerialPortButton">
               <property name="contextMenuPolicy">
                <enum>Qt::DefaultContextMenu</enum>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>扫描串口</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="openSerialButton">
               <property name="text">
                <string>打开串口</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="RecGroupBox">
            <property name="title">
             <string>接收设置</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,1,1">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QRadioButton" name="RecRadioASCII">
                 <property name="text">
                  <string>ASCII</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="RecRadioHex">
                 <property name="text">
                  <string>HEX</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QCheckBox" name="newlineCheckBox">
               <property name="text">
                <string>自动换行</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3,1">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>接收间隔     </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="RecTimerSpinBox">
                 <property name="minimum">
                  <number>0</number>
                 </property>
                 <property name="maximum">
                  <number>1000000000</number>
                 </property>
                 <property name="singleStep">
                  <number>10</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                 <property name="displayIntegerBase">
                  <number>10</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string> MS </string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="clearRecButton">
               <property name="text">
                <string>清除接收区信息</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="SendGroupBox">
            <property name="title">
             <string>发送设置</string>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5" stretch="1,1,1,1">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_5">
               <item>
                <widget class="QRadioButton" name="WriRadioASCII">
                 <property name="text">
                  <string>ASCII</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="WriRadioHex">
                 <property name="text">
                  <string>HEX</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QCheckBox" name="AutoCheckBox">
                 <property name="text">
                  <string>自动发送</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="ViewSendCheckBox">
                 <property name="text">
                  <string>显示发送</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3,1">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>发送间隔      </string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="SendTimeSpinBox">
                 <property name="minimum">
                  <number>1</number>
                 </property>
                 <property name="maximum">
                  <number>100000000</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>  MS</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QPushButton" name="clearSendButton">
               <property name="text">
                <string>清除发送区信息</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="viewWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="5,1">
       <item>
        <widget class="QGroupBox" name="viewMessgroupBox">
         <property name="title">
          <string>接收区</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QTextBrowser" name="textBrowser"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="sendMessgroupBox">
         <property name="title">
          <string>发送区</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QTextEdit" name="textEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="SendSerialButton">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBlackTea">
    <property name="title">
     <string>BlackTea</string>
    </property>
    <addaction name="actionCSU"/>
   </widget>
   <addaction name="menuBlackTea"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionCSU">
   <property name="icon">
    <iconset>
     <normaloff>icon/university.png</normaloff>icon/university.png</iconset>
   </property>
   <property name="text">
    <string>CSU</string>
   </property>
  </action>
  <action name="actionSerialState">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/icon/redled.png</normaloff>:/icon/icon/redled.png</iconset>
   </property>
   <property name="text">
    <string>当前串口</string>
   </property>
   <property name="toolTip">
    <string>当前串口状态</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/icon/redled.png</normaloff>:/icon/icon/redled.png</iconset>
   </property>
   <property name="text">
    <string>接收信号</string>
   </property>
   <property name="toolTip">
    <string>收到数据闪烁</string>
   </property>
  </action>
  <action name="actionSendData">
   <property name="icon">
    <iconset>
     <normaloff>:/icon/icon/redled.png</normaloff>:/icon/icon/redled.png</iconset>
   </property>
   <property name="text">
    <string>发送数据</string>
   </property>
   <property name="toolTip">
    <string>发送数据闪烁</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
